nmostesting/GenericTest.py

```diff
@@ -390,20 +406,34 @@ def basics(self):
             # Perform an automatic check for an error condition
             results.append(self.do_test_404_path(api))
 
+            # Test that the API responds with a 4xx when a missing or invalid token is used
```

Comment:
I would suggest a test for https://github.com/AMWA-TV/nmos-authorization/pull/67 in this area

Reply:
Yes, now we've resolved that I'll see if I can straightforwardly add something.

Reply:
I've added some testing for that (and removed a test which didn't make sense anymore). This could be expanded later to cover all cases from the table.